ts/utils/navigation.ts

```diff
@@ -19,3 +20,10 @@ export const IO_INTERNAL_LINK_PREFIX = IO_INTERNAL_LINK_PROTOCOL + "//";
 
 export const convertUrlToNavigationLink = (path: string) =>
   path.replace(IO_INTERNAL_LINK_PREFIX, "/");
+
+/**
+ * This variable should be used on every `gestureEnabled` setting
+ * in the navigator. This prevents the gestures to be enabled on Android
+ * creating glitches with the scroll on old Android versions (version 9 and below).
+ */
+export const areGestureEnabled = Platform.OS !== "android";
```

Comment:
Please, don't hate me, but I would call it `isGestureEnabled` (otherwise `areGesturesEnabled`) 😅

Reply:
You're 1000000% right, I missed this when I renamed it from `areGesturesEnabled` 😂  It should be fixed here: https://github.com/pagopa/io-app/pull/4094/commits/e0be7201aa78bfd3372e85ee18a21002435fc8b1